index.csv

```diff
@@ -1,11 +1,12 @@
+acorn-jsx,MIT,0.6.1,1.0.0,1.0.1,1.0.2,1.0.3,2.0.0,2.0.1,3.0.0,3.0.1
```

Comment:
cc @rreverser @adrianheine @marijn

Reply:
Previous versions used `licenses` declaration, which is now deprecated, but was still correct SPDX declaration at the time. https://unpkg.com/browse/acorn-jsx@1.0.0/package.json

Might be worth updating the script you used for collecting the data to count this old style as valid too.